requirements.in

```diff
@@ -1,5 +1,5 @@
 https://github.com/codecov/opentelem-python/archive/refs/tags/v0.0.4a1.tar.gz#egg=codecovopentelem
-https://github.com/codecov/shared/archive/f9b9ecab5c927f6118dc7fa473f45a5b07d7e581.tar.gz#egg=shared
```

Comment:
this is the commit previous to [mine](https://github.com/codecov/shared/pull/335) in shared